devnet-1/README.md

```diff
@@ -0,0 +1,118 @@
+# Run celestia-app & core
+
+## clone the repo and install
+```sh
+git clone https://github.com/celestiaorg/lazyledger-app.git
+
+cd lazyledger-app 
+
+make install
+
+# check that celestia-appd --help is working properly
+celestia-appd --help
+```
+
+## Initialize your validator
+
+```sh
+celestia-appd config chain-id devnet-1
+
+# provide network and moniker name (i.e. name of your validator -- choose any name!)
+celestia-appd init <moniker_name> --chain-id devnet-1
+```
+This will create a new `.celestia-app` folder in your HOME directory.
+
+### Download Pregenesis File
+
+You can now download the "pregenesis" file for the chain.  This is a genesis file with the chain-id and airdrop balances.
+
+```sh
+cd $HOME/.celestia-app/config/
+
+curl https://raw.githubusercontent.com/celestiaorg/networks/master/devnet-1/pregenesis.json > $HOME/.celestia-app/config/genesis.json
+```
+
+
+## Get your validator pubkey 
+
+You must get your validator's consensus pubkey as it will be necessary to include in the transaction to create your validator.
+
+If you are using Tendermint's native `priv_validator.json` as your consensus key, you display your validator public key using the following command
+
+```
+celestia-appd tendermint show-validator
+```
+
+The pubkey should be formatted with the bech32 prefix `celesvalconspub1`.
+
+If you are using a custom signing mechanism such as `tmkms`, please refer to their relevant docs to retrieve your validator pubkey.
+
+
+### Create GenTx
+
+Now that you have you key imported, you are able to use it to create your gentx.
+
+To create the genesis transaction, you will have to choose the following parameters for your validator:
+
+- moniker
+- commission-rate
+- commission-max-rate
+- commission-max-change-rate
+- min-self-delegation (must be >1)
+- website (optional)
+- details (optional)
+- identity (keybase key hash, this is used to get validator logos in block explorers. optional)
+- pubkey (gotten in previous step)
```

Comment:
Most of these are not used below as far as I can see. We should make it consistent. We should check which are really optional and which aren't.